### PR TITLE
Let us add items to the `UrlRelative` enum without breaking

### PR DESCRIPTION
**Pull request by notriddle**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -620,6 +620,10 @@ pub enum UrlRelative<'a> {
     PassThrough,
     /// Relative URLs will be changed into absolute URLs, based on this base URL.
     RewriteWithBase(&'a str),
+    // Do not allow the user to exhaustively match on UrlRelative,
+    // because we may add new items to it later.
+    #[doc_hidden]
```
Comment (lnicola) [inline src/lib.rs]: #[doc(hidden)]

File: src/lib.rs [hunk 2]
```diff
@@ -620,6 +620,10 @@ pub enum UrlRelative<'a> {
...
+    _NonExhaustive,
```
Comment (Eijebong) [inline src/lib.rs]: I've only seen `__NonExhaustive` until now (with two `_`), I don't think it really matters, just saying :p

File: Cargo.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ammonia"
-version = "0.7.0" # remember to update html_root_url
+version = "0.7.0-rc1" # remember to update html_root_url
```
Comment (lnicola) [inline Cargo.toml]: Should this be `1.0-rc1` and in the other PR?

Comment (notriddle) [reply Cargo.toml]: Yes 🤦‍♂️

File: src/lib.rs (hunk 2 above)
Comment (notriddle) [reply src/lib.rs]: Fixed

File: src/lib.rs (hunk 1 above)
Comment (notriddle) [reply src/lib.rs]: Fixed

File: Cargo.toml (hunk 3 above)
Comment (lnicola) [reply Cargo.toml]: And umm.. "remember to update html_root_url". Does docs.rs work for `-rc` releases?

**Comment by notriddle** [comment]
I removed the version bump from this PR. It should be ready-to-go, then, even if the other ones aren't.

**Comment by lnicola** [comment]
Did I break bors? 😄 

bors retry

**Comment by notriddle** [comment]
No, I think Travis has just been having a bad time for the last month or so.

**Pull request merged**